tests/modules/images/webp-uploads/load-tests.php

```diff
@@ -918,11 +920,63 @@ public function it_should_create_mime_types_for_allowed_sizes_only_via_global_va
 		$this->assertImageNotHasSizeSource( $attachment_id, 'not_allowed_size_200x150', 'image/webp' );
 	}
 
+	/**
+	 * Test image quality for image conversion.
+	 *
+	 * @test
+	 */
+	public function test_set_quality_with_image_conversion() {
```

Comment:
Remove `test_` from the function name.
```suggestion
	public function it_should_set_quality_with_image_conversion() {
```

@mehulkaklotar We have two options for running a test.

- At starting of function name use `test_`
- Or you can use the `@test` annotation in a method’s docblock.